### ISSUE TITLE
Problem to update watir-rails from version 0.1.2 to 1.x

### ISSUE THREAD

**Issue by severinedarlot** [watir/watir-rails#16]
Hello, I have problem when I try to update watir-rails from version 0.1.2 to 1.1.0. This problem is the same for the version 1.0.0.

When I update my gemfile
```
group :test do
  [...]
  gem 'watir', '~> 5.0'
  gem 'watir-rails', '1.1.0'
  gem 'watir-rspec'
  gem 'watir-webdriver'
  [...]
end
```
bundle command gives:
```
[...]
Using watir-webdriver 0.9.1
Using watir 5.0.0
Using watir-rails 1.1.0
Using watir-rspec 2.0.3
[...]
```
But when I run my tests, the url in the browser is wrong because it misses the rails host and the rails port. To understand why, I debugged and saw that when the test runs the `goto "/app" line` the method Watir::Browser::goto of watir-rails gem is not called. In fact, the method Watir::Browser::goto of watir-webdriver is called:

```
From: /home/darlot/.rvm/gems/ruby-2.2.2/gems/watir-webdriver-0.9.1/lib/watir-webdriver/browser.rb @ line 75 Watir::Browser#goto:

    74: def goto(uri)
 => 75:   uri = "http://#{uri}" unless uri =~ URI.regexp
    76: 
    77:   @driver.navigate.to uri
    78:   @after_hooks.run
    79: 
    80:   uri
    81: end

[1] pry(#<Watir::Browser>):1> 

```

So I guess I have a gem loading order problem. So I tried to change the order of the gem in the gemfile:

```ruby
group :test do
 [...]
  gem 'watir-webdriver'
  gem 'watir', '~> 5.0'
  gem 'watir-rails', '1.1.0'
  gem 'watir-rspec'
 [...]
end
```

But this time, I obtained a infinite recursion when I run my test. Here is the console log:

```
# /home/sdarlot/.rvm/gems/ruby-2.2.2/gems/watir-rails-1.1.0/lib/watir/browser.rb:53:in `initialize_rails_with_watir'
     # /home/sdarlot/.rvm/gems/ruby-2.2.2/gems/watir-rails-1.1.0/lib/watir/browser.rb:10:in `initialize'
     # /home/sdarlot/.rvm/gems/ruby-2.2.2/gems/watir-5.0.0/lib/watir/loader.rb:9:in `initialize'
     # /home/sdarlot/.rvm/gems/ruby-2.2.2/gems/watir-rails-1.1.0/lib/watir/browser.rb:53:in `block in initialize_rails_with_watir'
     # /home/sdarlot/.rvm/gems/ruby-2.2.2/gems/watir-rails-1.1.0/lib/watir/browser.rb:36:in `call'
     # /home/sdarlot/.rvm/gems/ruby-2.2.2/gems/watir-rails-1.1.0/lib/watir/browser.rb:36:in `override_and_preserve_original_methods'
     # /home/sdarlot/.rvm/gems/ruby-2.2.2/gems/watir-rails-1.1.0/lib/watir/browser.rb:53:in `initialize_rails_with_watir'
     # /home/sdarlot/.rvm/gems/ruby-2.2.2/gems/watir-rails-1.1.0/lib/watir/browser.rb:10:in `initialize'
     # /home/sdarlot/.rvm/gems/ruby-2.2.2/gems/watir-5.0.0/lib/watir/loader.rb:9:in `initialize'
     # /home/sdarlot/.rvm/gems/ruby-2.2.2/gems/watir-rails-1.1.0/lib/watir/browser.rb:53:in `block in initialize_rails_with_watir'
     # /home/sdarlot/.rvm/gems/ruby-2.2.2/gems/watir-rails-1.1.0/lib/watir/browser.rb:36:in `call'
     # /home/sdarlot/.rvm/gems/ruby-2.2.2/gems/watir-rails-1.1.0/lib/watir/browser.rb:36:in `override_and_preserve_original_methods'
     # /home/sdarlot/.rvm/gems/ruby-2.2.2/gems/watir-rails-1.1.0/lib/watir/browser.rb:53:in `initialize_rails_with_watir'
     # /home/sdarlot/.rvm/gems/ruby-2.2.2/gems/watir-rails-1.1.0/lib/watir/browser.rb:10:in `initialize'
     # ./spec/spec_helper.rb:85:in `new'
     # ./spec/spec_helper.rb:85:in `create_browser'
     # ./spec/spec_helper.rb:177:in `block (2 levels) in <top (required)>'
```

Do you have a clue to help me solving the watir-rails update ?
Thanks

**Comment by jarmo** [comment]
Following is totally untested, but as far as I remember then you should only include `watir-rails` and `watir-rspec` in your `Gemfile`, because both of them require all necessary gems in correct order.

In your `spec_helper.rb` just require `watir/rails` and `watir/rspec` and then set your `Watir.driver` to `:webdriver` or create a `Browser` object with a specific browser (e.g. `Watir::Browser.new :firefox`).

As far as I know then order of gems in `Gemfile` does not matter when it comes to loading gems.

**Comment by severinedarlot** [comment]
Thanks for the strick. It works !
I've just removed `gem watir` and `gem 'watir-webdriver'` from my Gemfile and I've added `require 'watir/rails'` and `require 'watir/rspec'` in the spec_helper.rb file.

I have just a little problem with the line `Watir.default_timeout = 3` in my spec_helper.rb file because it requires watir-webdriver. But if I add `require 'watir/webdriver'` I obtain the same problem of loading.
